### PR TITLE
Bug/dcd 314 315 analytics header linnk

### PR DESCRIPTION
**Pull request by fazanki**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: app/views/includes/scripts_ga_1.html [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-        j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-        'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-        })(window,document,'script','dataLayer','GTM-KG8XT3S');
-    </script>
-    <!-- End Google Tag Manager -->
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+})(window,document,'script','dataLayer','GTM-KG8XT3S');</script>
+<!-- End Google Tag Manager -->
 {% endif %}
```
Comment (LucaDelBuonoHMCTS) [inline app/views/includes/scripts_ga_1.html:9]: Undo all of this.

File: app/views/includes/scripts_ga_2.html [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
-    <!-- Google Tag Manager (noscript) -->
-    <noscript>
-        <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KG8XT3S" height="0" width="0" style="display:none;visibility:hidden" title="gtm"></iframe>
-    </noscript>
-    <!-- End Google Tag Manager (noscript) -->
+<!-- Google Tag Manager (noscript) -->
+<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-KG8XT3S"
+                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+<!-- End Google Tag Manager (noscript) -->
 {% endif %}
```
Comment (LucaDelBuonoHMCTS) [inline app/views/includes/scripts_ga_2.html:6]: Undo all of this.

**Review by LucaDelBuonoHMCTS — CHANGES_REQUESTED** [review]

File: app/views/includes/scripts_ga_1.html (hunk 1 above)
Comment (fazanki) [reply app/views/includes/scripts_ga_1.html:11]: done

File: app/views/includes/scripts_ga_2.html (hunk 2 above)
Comment (fazanki) [reply app/views/includes/scripts_ga_2.html:7]: done

File: app/views/includes/scripts_ga_1.html (hunk 1 above)
Comment (LucaDelBuonoHMCTS) [reply app/views/includes/scripts_ga_1.html:11]: Did you push? I still see these changes.

File: app/views/includes/scripts_ga_2.html (hunk 2 above)
Comment (LucaDelBuonoHMCTS) [reply app/views/includes/scripts_ga_2.html:7]: Did you push? I still see these changes.

**Review by LucaDelBuonoHMCTS — APPROVED** [review]

**Pull request closed without merging**